src/rt-app.c

```diff
@@ -1035,6 +1039,7 @@ void *thread_body(void *arg)
 
 		set_thread_affinity(data, &pdata->cpu_data);
 		set_thread_priority(data, pdata->sched_data);
+		set_thread_taskgroup(data, pdata->taskgroup_data);
```

Comment:
Here is handling the taskgroup set like changing sched policy or affinity, right?

Reply:
Yes, that's right.